app/api/v2/db_config.py

```diff
@@ -30,39 +31,76 @@ def create_tables():
         cur.execute(q)
     conn.commit()
 
+
 def drop_all():
     """
     Method to drop all the tables in the database
     """
+    cur.execute("DROP TABLE IF EXISTS categories CASCADE")
     cur.execute("DROP TABLE IF EXISTS users CASCADE")
     cur.execute("DROP TABLE IF EXISTS stores CASCADE")
     cur.execute("DROP TABLE IF EXISTS products CASCADE")
     cur.execute("DROP TABLE IF EXISTS carts CASCADE")
     cur.execute("DROP TABLE IF EXISTS sales CASCADE")
     conn.commit()
 
+
 def tables():
     """
     This function defines the different tables in the database
     It returns all the definitions in a queries list
     """
 
-    stores =  """
-        CREATE TABLE IF NOT EXISTS stores(id serial PRIMARY KEY, 
-        name varchar, 
+    stores = """
+        CREATE TABLE IF NOT EXISTS stores(id serial PRIMARY KEY,
+        name varchar,
         category varchar,
         created_at varchar);
         """
 
     users = """
-        CREATE TABLE IF NOT EXISTS users(id serial PRIMARY KEY, 
+        CREATE TABLE IF NOT EXISTS users(id serial PRIMARY KEY,
         store_id int,
         role int,
-        email varchar, 
+        email varchar,
         password varchar,
         added_at varchar);
         """
 
-    queries = [stores,users]
+    carts = """
+        CREATE TABLE IF NOT EXISTS carts(id serial PRIMARY KEY,
+        seller_id int,
+        product varchar,
+        number int,
+        amount int,
+        created_at varchar);
+        """
+
+    sales = """
+        CREATE TABLE IF NOT EXISTS sales(id serial PRIMARY KEY,
+        store_id int,
+        seller_id int,
+        product varchar,
+        number int,
+        amount int,
+        created_at varchar);
+        """
+
+    products = """
+        CREATE TABLE IF NOT EXISTS products(id serial PRIMARY KEY,
+        store_id int,
+        name varchar,
+        inventory int,
+        price int,
+        category varchar,
+        created_at varchar);
+        """
+    categories = """
+        CREATE TABLE IF NOT EXISTS categories(id serial PRIMARY KEY,
+        store_id int,
+        name varchar,
+        created_at varchar);
+        """
+
+    queries = [stores, users, products, sales, carts, categories]
     return queries
-    
```

Comment:
Leave an empty line at the bottom of the file.

Reply:
This file has refused a new line
But I will keep trying